modules/govuk/manifests/node/s_base.pp

```diff
@@ -109,4 +110,8 @@
   user { 'ubuntu':
     ensure => absent,
   }
+
+  govuk_bundler::config { 'deploy-bundler':
```

Comment:
If the default of the user_home is related with the deploy user, I would put this resource next to the 'deploy' user, that I think it is govuk::deploy::setup, and make sure there is a require that ensures /home/deploy exists before creating this resource